scale-tests/batch_test.py

```diff
@@ -123,10 +128,28 @@ def submit_loop(submits_per_min: int, dispatchers: typing.List[typing.Dict], use
                  "--conf", "spark.mesos.driver.failoverTimeout={}".format(args["--spark-mesos-driver-failover-timeout"])
                 ]
 
+    if args["--spark-mesos-executor-gpus"]:
+        MEMORY_MULTIPLIER = 20
+        memory = int(args["--spark-mesos-executor-gpus"]) * MEMORY_MULTIPLIER
+        user_conf += ["--conf", "spark.driver.memory={}g".format(str(memory)),
```

Comment:
Some questions:
* Why does the memory depend on the number of GPUs?
* Hoe was the `MEMORY_MULTIPLIER` determined?
* Should the driver and executor memory be the same?
* Should we check that `spark.mesos.gpus.max` and `spark.mesos.executor.gpus` are consistent?
* There is no check that these args are specified if required (especially w.r.t `--docker-image` which has no default value).

Reply:
> What do you mean by consistent? That max is always >= executor.gpus?
Yes. It's not critical though.